### PR TITLE
Create is-prime function

### PR DESCRIPTION
**Pull request by EvandroLG**
I added a function to check if a given number is prime or not.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1486,6 +1487,21 @@ clamp(3, false, 8); // throws
...
 ### [just-modulo](https://www.npmjs.com/package/just-modulo)
 
+:icecream:[`Try It`](https://anguscroll.com/just/just-is-prime)
+
+`npm install just-is-prime`
+
+```js
+import modulo from "just-is-prime";
+
+isPrime(1); // false
```
Comment (angus-c) [inline README.md:1497]: can you add examples for a) non integer numbers b) other types c) null/undefined, all of which should throw
(similar to https://github.com/angus-c/just/blob/master/packages/array-median/README.md#L13)

File: packages/number-is-prime/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Guilt-free utilities for every occasion.
+
+```
+  import isPrime from 'just-is-prime;
+
+/*
+  isPrime(1); // false
+  isPrime(2); // true
+  isPrime(17); // true
+  isPrime(10); // false
```
Comment (angus-c) [inline packages/number-is-prime/README.md:13]: same as other README

File: packages/number-is-prime/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+/*
+  isPrime(1); // false
+  isPrime(2); // true
+  isPrime(17); // true
+  isPrime(10); // false
+*/
+
+function isPrime(number) {
+  if (!Number.isInteger(number)) {
+    throw new Error('Expected an integer');
```
Comment (angus-c) [inline packages/number-is-prime/index.js:12]: `just-prime` expects an integer argument

File: packages/number-is-prime/index.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  }
+
+  if (number < 2) {
+    return false;
+  }
+
+  for (var i = 2; i < Math.sqrt(number); i++) {
+    if (number % i === 0) return false;
```
Comment (angus-c) [inline packages/number-is-prime/index.js:20]: house style is curly braces for all `if`s

File: test/number-is-prime/index.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+var test = require('../util/test')(__filename);
+var isPrime = require('../../packages/number-is-prime');
+
+test('number is prime', function(t) {
```
Comment (angus-c) [inline test/number-is-prime/index.js:4]: also test for `0` and one large number, non integer numbers, other types, null/undefined

**Review by angus-c — CHANGES_REQUESTED** [review]
Thanks this is great! Just a few nits

File: README.md [hunk 6]
```diff
@@ -1486,6 +1487,27 @@ clamp(3, false, 8); // throws
 
 ### [just-modulo](https://www.npmjs.com/package/just-modulo)
 
+:icecream:[`Try It`](https://anguscroll.com/just/just-is-prime)
+
+`npm install just-is-prime`
+
+```js
+import modulo from "just-is-prime";
```
Comment (angus-c) [inline README.md:1495]: typo: isPrime not modulo

File: test/number-is-prime/index.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  t.notOk(isPrime(0));
+  t.notOk(isPrime(1));
+  t.notOk(isPrime(10));
+  t.notOk(isPrime(-10));
+  t.notOk(isPrime(13440));
+  t.end();
+});
+
+test('throw error', function(t) {
+  t.throws(function() {
```
Comment (angus-c) [inline test/number-is-prime/index.js:23]: could add a non-integer number here too

**Review by angus-c — CHANGES_REQUESTED** [review]
Sorry for late response! This looks great, just a couple of very small things. After that I will try to land and publish tonight

File: test/number-is-prime/index.js (hunk 7 above)
Comment (EvandroLG) [reply test/number-is-prime/index.js:23]: @angus-c I'm testing against `undefined`, `function`, `string`, `object`, etc.

**Review by angus-c — APPROVED** [review]

**Comment by angus-c** [comment]
Thanks!
